totalopenstation/tests/test_leica_gsi.py

```diff
@@ -48,4 +48,4 @@ def test_linestring(self):
         self.ls = self.fp.build_linestring()
         self.assertAlmostEqual(self.ls.coords[0][0], 515.836)
         self.assertAlmostEqual(self.ls.coords[3][2], 2.553)
-        self.assertAlmostEqual(self.ls.bounds[0], -7.4662080)
+        self.assertAlmostEqual(self.ls.bounds[0], -7.4660913)
```

Comment:
This is a slight change, but do we have a way to check what it the correct value without adjusting tests? E.g. with known sample data that is processed by another "trusted" software.

Reply:
I do not know what is your expectation on precision.
I think mm for metric is the minimum and it is really difficult to obtain a precision less than that during a survey or stakeout.